cmd/root.go

```diff
@@ -52,6 +52,7 @@ provide 0 to never give up or a negative number to disable`)
 	cmd.Flags().DurationVarP(&flags.WaitAndRetry, "retry-wait", "w", 3*time.Second, "time to wait before trying to reconnect to ssh server")
 	cmd.Flags().StringVarP(&flags.SshAgent, "ssh-agent", "A", "", "unix socket to communicate with a ssh agent")
 	cmd.Flags().DurationVarP(&flags.Timeout, "timeout", "t", 3*time.Second, "ssh server connection timeout")
+	cmd.Flags().StringVarP(&flags.Config, "config", "c", "", "set config file path")
```

Comment:
Can we have the default path configured as the default value of the config flag? Then maybe the `NewDefaultSSHConfigFilePath` function would be necessary anymore.

Reply:
Am I supposed to obtain home directory here? I've merged NewDefaultSSHConfigFilePath into NewSSHConfigFilePath

Reply:
You can set the default value to something like `$HOME/.ssh/config` and replace `$HOME` with the actual user home dir path.